lib/modules/userTagger.js

```diff
@@ -417,6 +490,19 @@ function populateDialog(tag: Tag, card) {
 	}
 
 	elements.openLink.addEventListener('click', () => openNewTabs('none', ...elements.link.value.split(/\s/)));
+	elements.presetTag.addEventListener('click', (e: MouseEvent) => {
+		const preset = {
```

Comment:
A simpler solution would be to add a listener when generating the preset option. Perhaps something like
```
elements.presetTag.append(...presetTags.map(([text, color]) => {
  const element = Tag.buildTagElement({ text, color });
  element.addEventListener('click', () => {
    tag.load({ text, color });
    tag.save();
    card.close();
  });
})
```